lib/workers/repository/update/branch/get-updated.spec.ts

```diff
@@ -235,7 +235,7 @@ describe('workers/repository/update/branch/get-updated', () => {
       expect(composer.updateArtifacts).toHaveBeenCalledWith(
         expect.objectContaining({
           config: expect.objectContaining({
-            lockFiles: ['composer.lock'],
+            lockFiles: ['different.lock'],
```

Comment:
This changes the intention of this test. In that case, it can be removed all-together. 
The code it tests works on real data, but testing it proved to be _wonky_.

Reply:
ref: https://github.com/renovatebot/renovate/pull/27319
Here this test curiously works.

Reply:
```suggestion
            lockFiles: ['composer.lock'],
```